app/scenes/SceneBrowser.js

```diff
@@ -246,7 +247,7 @@ SceneSceneBrowser.loadDataRequest = function() {
             theUrl = 'https://api.twitch.tv/kraken/users/' + encodeURIComponent(Config.data.userid) + '/follows/channels?sortby=last_broadcast&direction=desc&limit=' + SceneSceneBrowser.ItemsLimit + '&offset=' + offset;
         }
         else {
-            theUrl = 'https://api.twitch.tv/kraken/streams?limit=' + SceneSceneBrowser.ItemsLimit + '&offset=' + offset;
+            theUrl = 'https://api.twitch.tv/kraken/streams?language=pt-br&limit=' + SceneSceneBrowser.ItemsLimit + '&offset=' + offset;
```

Comment:
Remove `language=pt-br&` I think most users of this APP want English atm.
But it could be added as a language support feature if you are upp for it, but do a new branch/pull-request for that.